sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,8 +1,12 @@
 # Azure Form Recognizer client library for Java
-Form Recognizer is a cloud-based cognitive service that uses machine learning technology to identify and extract text, 
-key-value pairs and table data from form documents.
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize text and table data
+from form documents. Form Recognizer is made up of the following services:
```

Comment:
Apologies in advance because I've made changes to my readme since you looked at it, so I'll comment as much as I can the changes I've made.

Talking to @annelo-msft, she doesn't think they'll like saying that they're made up of services, should say something mroe like "...has the following functionalities"